pkg/controller/jenkins/reconciler.go

```diff
@@ -211,6 +214,13 @@ func (r *JenkinsReconciler) checkResourceIfExists(resource RuntimeResource) erro
 	return err
 }
 
+func (r *JenkinsReconciler) isPersistent() bool {
+	if r.ControlledRescources.JenkinsInstance.Spec.Persistence.Enabled {
+		return true
+	}
+	return false
+}
+
```

Comment:
You can simplify this into `return  r.ControlledRescources.JenkinsInstance.Spec.Persistence.Enabled`

Reply:
updated